Makefile

```diff
@@ -62,6 +64,12 @@ cross: $(foreach platform, $(SUPPORTED_PLATFORMS), $(BUILD_DIR)/$(RESOLVE_TAGS_P
 $(BUILD_DIR)/$(RESOLVE_TAGS_PROJECT)-%-$(GOARCH): $(GO_FILES) $(BUILD_DIR)
 	GOOS=$* GOARCH=$(GOARCH) CGO_ENABLED=0 go build -ldflags $(GO_LD_RESOLVE_FLAGS) -tags $(GO_BUILD_TAGS) -o $@ $(RESOLVE_TAGS_PACKAGE)
 
+.PHONY: cross-tar
+cross-tar: $(foreach platform, $(SUPPORTED_PLATFORMS), $(BUILD_DIR)/$(RESOLVE_TAGS_PROJECT)-$(platform).tar)
+
+$(BUILD_DIR)/$(RESOLVE_TAGS_PROJECT)-%.tar: cross
+	tar -cf $@ -C $(RESOLVE_TAGS_PATH) plugin.yaml -C $(PWD)/out/ resolve-tags-$*
```

Comment:
minor nit: open-source folks will find it very strange to see an uncompressed .tar distributed in the wild. It is customary to specify a type of compression, such as by using -z to specify filtering the file through gzip to emit a .tar.gz file.

There is also -J to specify ".xz" format, but that is something I would only worry about if the resulting file is >100MB due to the extra CPU overhead.

Reply:
For sure, I'll open another PR with this change